Automattic/woocommerce-payments
# Update woopay shortcode CTA text

Fixes https://github.com/Automattic/woopay/issues/1639

#### Changes proposed in this Pull Request

Update WooPay CTA text to differ from C&C Block's CTA









#### Testing instructions





* Add a product to the cart
* Go to Shortcode checkout
* Enter a WooPay unregistered email
* WooPay registration checkbox should show as below

![SCR-20230214-jg9](https://user-images.githubusercontent.com/6216000/218848277-9f72fafb-597b-40bf-8d5b-e6799caed911.png)

* WooPay registration checkbox in C&C Blocks page should remain unchanged. 

-------------------


**Post merge**
